### PR TITLE
Feat: products screen

### PR DESCRIPTION
**Pull request by ViniciusKalach**
Create products screen + products edit and create modal

### CODE REVIEW

**Comment by nicolasgandrade** [comment]
Open the PR again but with develop branch as the destination

**Comment by ViniciusKalach** [comment]
I edited the pull-request and switched from master to develop

File: src/main/java/com/fp3/haras/view/screens/products/ProductsScreen.java [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+public class ProductsScreen extends javax.swing.JPanel {
+    private JPanel selectedPanel;
+    ProductsCreate pm;
+    
+    public ProductsScreen() {
+        initComponents();
+        
+        this.setBackground(Colors.PRIMARYBG);
+
```
Comment (nicolasgandrade) [inline src/main/java/com/fp3/haras/view/screens/products/ProductsScreen.java]: You can remove this empty line

File: src/main/java/com/fp3/haras/view/screens/products/ProductsScreen.java [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    private JPanel selectedPanel;
+    ProductsCreate pm;
+    
+    public ProductsScreen() {
+        initComponents();
+        
+        this.setBackground(Colors.PRIMARYBG);
+        
+        lblTitle.putClientProperty("FlatLaf.styleClass", "h00");    
+
```
Comment (nicolasgandrade) [inline src/main/java/com/fp3/haras/view/screens/products/ProductsScreen.java]: This one too

**Review by nicolasgandrade — CHANGES_REQUESTED** [review]
Fine for me. Just make those changes I commented so I can approve.
Good job.

**Comment by ViniciusKalach** [comment]
I erased the lines, changes met

**Pull request merged**